templates/maas/index.html

```diff
@@ -1,1060 +1,700 @@
+    <h3 class="p-heading--5"><a href="https://www.youtube.com/watch?v=sLADei_c9Qg">Bare metal Kubernetes hands on tutorial with MAAS and Juju</a></h3>
+    <p>Go from bare metal to a fully operational Kubernetes cluster. This hands-on video tutorial guides you through the entire process using MAAS and Juju.</p>
+    '
+    }
+    }
+    ]
+    },
+    {
+    "contents": [
+    {
+    "type": "description",
+    "item": {
+    "type": "html",
+    "content": '
+    <h3 class="p-heading--5"><a href="/maas/tour">MAAS product tour</a></h3>
+    <p>Our tutorials offer clear, step-by-step instructions and practical demonstrations to help you master MAAS efficiently and effectively, from initial setup to advanced deployment.</p>
+    '
+    }
+    }
+    ]
+    },
+    ]
+    },
+    ],) 
+  }}
+
+  {{ vf_basic_section(title={"text": "Enterprise support for MAAS"},
+    is_split_on_medium=true,
+    items=[
+    {
+    "type": "image",
+    "padding": "shallow",
+    "item": {
+    "aspect_ratio": "16-9",
+    "attrs": {
+    "src": "https://assets.ubuntu.com/v1/50faa077-enterprise_support.png",
+    "alt": ""
+    }
+    }
+    },
+    {
+    "type": "description",
+    "padding": "shallow",
+    "item": {
+    "type": "html",
+    "content": '
+    <p>MAAS is freely available, open source software from Canonical. Support and commercial capabilities are available.</p>
+    <h3 class="p-heading--5 u-no-padding u-no-margin">Ubuntu Pro with Infra-only coverage</h3>
+    <p>Get security, compliance, and support for full Ubuntu deployments managed by MAAS. This coverage is charged on a per-machine basis.</p>
+    '
+    }
+    },
+    {
+    "type": "cta-block",
+    "item": {
+    "primary": {
+    "content_html": "Learn more about Ubuntu Pro",
+    "attrs": {
+    "href": "https://ubuntu.com/pricing/pro"
+    }
+    }
+    }
+    }
+    ]) 
+  }}
+
+  {% call(slot) vf_pricing_block(
+    title_text='MAAS standalone support',
+    top_rule_variant="highlighted",
+    tiers=[
+    {
+    'tier_name_text': 'Self-support',
+    'tier_price_text': 'Free',
+    'tier_price_explanation': 'for all machines',
+    'tier_label_text': 'What’s included',
+    'tier_offerings': [
+    {
+    'list_item_style': 'crossed',
+    'list_item_content_html': 'High availability (HA) support'
+    },
+    {
+    'list_item_style': 'crossed',
+    'list_item_content_html': 'Role based access control (RBAC)'
+    },
+    {
+    'list_item_style': 'crossed',
+    'list_item_content_html': 'Knowledge Base'
+    },
+    {
+    'list_item_style': 'crossed',
+    'list_item_content_html': 'Response time - SLA Sev 1'
+    },
+    {
+    'list_item_style': 'crossed',
+    'list_item_content_html': 'Phone and ticket support'
+    }
+    ],
+    'cta_html': '<a href="/maas/docs/how-to-get-maas-up-and-running">Install MAAS ›</a>'
+    },
+
+    {
+    'tier_name_text': 'Essential',
+    'tier_price_text': '$30',
+    'tier_price_explanation': 'per year, per managed machine',
```

Comment:
In the design these explanations are indented when they wrap on to a new line to leave space for the price.

Reply:
Again, I assume that what's in the pattern is OK.